core/model/edition.py

```diff
@@ -724,6 +724,17 @@ def calculate_author(self):
             sort_author = " ; ".join(sorted(sort_names))
         else:
             sort_author = self.UNKNOWN_AUTHOR
+
+        def truncate_string(mystr: str):
+            if len(mystr) > 1024:
+                return mystr[: (1024 - 3)] + "..."
+            else:
+                return mystr
+
+        # Very long author and sort_author strings can cause the limits of one or more of the
+        # underlying postgres indices.  Therefore,  we limit their size here.
```

Comment:
Can you clean up this comment a bit? I think maybe you missed a word or two...

Maybe add a bit of info about the error we were seeing from PG.